optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeVariableCreationImportIT.java

```diff
@@ -126,6 +126,37 @@ public void variableImportWorksForLongStrings() {
             });
   }
 
+  @Test
+  public void variableImportWorksForDateStrings() {
+    // given
+    final Process deployedProcess = zeebeExtension.deployProcess(createStartEndProcess(PROCESS_ID));
+    final String variableName = "date";
+    final String variableValue = "2025-10-10";
+    final String parsedDateValue = "2025-10-10T00:00:00.000+0000";
+    final Map<String, Object> variables = Map.of(variableName, variableValue);
+    final Long processInstanceKey =
+        zeebeExtension.startProcessInstanceWithVariables(
+            deployedProcess.getBpmnProcessId(), variables);
+
+    // when
+    waitUntilNumberOfDefinitionsExported(1);
+    waitUntilMinimumProcessInstanceEventsExportedCount(4);
+    waitUntilMinimumVariableDocumentsExportedCount(1);
+    importAllZeebeEntitiesFromScratch();
+
+    // when
+    final ProcessInstanceDto importedProcessInstance =
+        getProcessInstanceForId(String.valueOf(processInstanceKey));
+    assertThat(importedProcessInstance.getVariables())
+        .singleElement()
+        .satisfies(
+            variable -> {
+              assertThat(variable.getName()).isEqualTo(variableName);
+              assertThat(variable.getType()).isEqualTo(DATE.getId());
+              assertThat(variable.getValue().getFirst()).isEqualTo(parsedDateValue);
```

Comment:
🔧 I guess there will be only one value:

```suggestion
              assertThat(variable.getValue()).containsExactly(parsedDateValue);
```